classes/ValidateConstraintTranslator.php

```diff
@@ -49,14 +49,32 @@ public function __construct(TranslatorInterface $translator)
      */
     public function translate($validator)
     {
-        if ($validator === 'isName' || $validator === 'isCustomerName') {
+        if ($validator === 'isName') {
             return $this->translator->trans(
                 'Invalid name',
                 [],
                 'Shop.Forms.Errors'
             );
         }
 
+        if ($validator === 'isCustomerName') {
+            return $this->translator->trans(
+                'Invalid name',
+                [],
+                'Shop.Forms.Errors'
+            ) . PHP_EOL .
+            $this->translator->trans(
+                'The following characters are not accepted: / \ ^ * `',
```

Comment:
I think we should tend towards shorter wordings, otherwise error notifications will be too present and discouraging. What about _Invalid characters: /\^*`_ instead?

Reply:
Seems good to me, @samuel-pires wdyt?